docs/iris/src/further_topics/metadata.rst

```diff
@@ -0,0 +1,987 @@
+
+We can inspect their associated ``metadata``,
+
+.. doctest:: richer-metadata
+
+    >>> forecast_period.metadata
+    CoordMetadata(standard_name='forecast_period', long_name=None, var_name='forecast_period', units=Unit('hours'), attributes={}, coord_system=None, climatological=False)
+    >>> latitude.metadata
+    DimCoordMetadata(standard_name='latitude', long_name=None, var_name='latitude', units=Unit('degrees'), attributes={}, coord_system=GeogCS(6371229.0), climatological=False, circular=False)
+
+Before comparing them to determine the values of metadata members that are different,
+
+.. doctest:: richer-metadata
+
+    >>> forecast_period.metadata.difference(latitude.metadata)
+    CoordMetadata(standard_name=('forecast_period', 'latitude'), long_name=None, var_name=('forecast_period', 'latitude'), units=(Unit('hours'), Unit('degrees')), attributes=None, coord_system=(None, GeogCS(6371229.0)), climatological=None)
+
+.. doctest:: richer-metadata
+
+    >>> latitude.metadata.difference(forecast_period.metadata)
+    DimCoordMetadata(standard_name=('latitude', 'forecast_period'), long_name=None, var_name=('latitude', 'forecast_period'), units=(Unit('degrees'), Unit('hours')), attributes=None, coord_system=(GeogCS(6371229.0), None), climatological=None, circular=(False, None))
+
+In general, however, comparing **different** metadata classes will result in a
+``TypeError`` being raised,
+
+.. doctest:: richer-metadata
+
+    >>> cube.metadata.difference(longitude.metadata)
+    Traceback (most recent call last):
+    TypeError: Cannot differ 'CubeMetadata' with <class 'iris.common.metadata.DimCoordMetadata'>.
+
+
+.. _metadata combine:
+
+Metadata combination
+^^^^^^^^^^^^^^^^^^^^
+
+.. testsetup:: metadata-combine
+
+   import iris
+   cube = iris.load_cube(iris.sample_data_path("A1B_north_america.nc"))
+   longitude = cube.coord("longitude")
+
+So far we've seen how to :ref:`compare metadata <metadata equality>`, and also how
+to determine the :ref:`difference between metadata <metadata difference>`. Now we
+take the next step, and explore how to combine metadata together using the ``combine``
+metadata class method.
+
+For example, consider the following :class:`~iris.common.metadata.CubeMetadata`,
+
+.. doctest:: metadata-combine
+
+    >>> cube.metadata  # doctest: +SKIP
+    CubeMetadata(standard_name='air_temperature', long_name=None, var_name='air_temperature', units=Unit('K'), attributes={'Conventions': 'CF-1.5', 'STASH': STASH(model=1, section=3, item=236), 'Model scenario': 'A1B', 'source': 'Data from Met Office Unified Model 6.05'}, cell_methods=(CellMethod(method='mean', coord_names=('time',), intervals=('6 hour',), comments=()),))
+
+We can perform the **identity function** by comparing the metadata with itself,
+
+.. doctest:: metadata-combine
+
+    >>> metadata = cube.metadata.combine(cube.metadata)
+    >>> cube.metadata == metadata
+    True
+
+As you might suspect, combining identical metadata returns metadata that is
```

Comment:
Q:  Do you mean "As you might expect" rather that "as expected" because, one might assume the metadata gets smashed together?  (As in leinient data? - I don't know as I've not read that section yet).